integrations/jfrog.rst

```diff
@@ -0,0 +1,74 @@
+.. _integrations_jfrog:
+
+|jfrog_logo| JFrog
+==================
+
+Artifactory Build Info
+----------------------
+
+.. warning::
+
+    The support of Artifactory Build Info via extension commands is not guaranteed as stable yet.
+    See :ref:`the Conan stability<stability>` section for more information.
+
+The `Artifactory build info <https://www.buildinfo.org/>`_ is a recollection of the metadata of a build.
+This json-formatted file includes all the details about the build broken down into segments like version history, artifacts, project
+modules, dependencies, and everything that was required to create the build.
+
+Build infos are identified with a ``build name`` and a ``build number``, similar to how many CI services identify the builds.
+They are conveniently stored in Artifactory to keep track of the build metadata to later perform different operations.
+
+Conan does not offer built-in support for the build info format. However, we have developed some :ref:`custom commands<reference_commands_custom_commands>` at
+at the `extensions repository <https://github.com/conan-io/conan-extensions>`_ using the feature, that provides support to create and manage the build info files.
+
+How to install the build info extension commands
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+Using the dedicated repository for Conan extensions https://github.com/conan-io/conan-extensions, it is as easy as:
+
+.. code-block:: bash
+
+    $ conan config install https://github.com/conan-io/conan-extensions
```

Comment:
Instead of a note, can we better just install the extensions for JFrog only? with ``-sf=artifactory -tf=artifactory`` or something like that? I wouldn't like to pollute the installation of users with many other unrelated commands.